workbench/static/script/paging.js

```diff
@@ -0,0 +1,66 @@
+function Paging(table_name, num_items_in_page) {
```

Comment:
@kevinkuan why are there 2 paging.js files in the review?

Reply:
Hmm, both paging.js are identical, but it doesn't seem like the one located at workbench/paging.js is doing anything. I overlooked this, I'll go ahead and remove the unused one.